packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -552,7 +553,7 @@ export const actionCreators: ActionCreators = {
         const createResp = await getDevWorkspaceClient().createDevWorkspace(
           defaultNamespace,
           devWorkspaceResource,
-          editorId,
+          cheEditor,
```

Comment:
Is it ok to make this change @olexii4 ?

I noticed that this section was changed in this PR: https://github.com/eclipse-che/che-dashboard/pull/922/files#diff-c9344a8d3b76b9a96206044091ed1fff4924dbd53684a56ad0f8cb02a80f04fc

Reply:
@dkwon17 It is ok to make this change.